### PR TITLE
[#687] Add Sentry

### LINKED ISSUE #687: Add Sentry

### PR DESCRIPTION
**Pull request by kmarszalek**
Closes #687.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -44,6 +44,7 @@
         <unirest.version>3.10.00</unirest.version>
         <bucket4j-starter.version>0.2.0</bucket4j-starter.version>
         <caffeine.version>2.8.5</caffeine.version>
+        <sentry.version>1.6.3</sentry.version>
```
Comment (jswk) [inline pom.xml:47]: Add a comment here not to update this, unless Sentry instance was updated. I will forget about not upgrading this otherwise

Comment (kmarszalek) [reply pom.xml:50]: but i will not! and you need approve :P

Comment (jswk) [reply pom.xml:50]: Okay, in such cases I will blame you! :D

File: s4e-backend/pom.xml [hunk 2]
```diff
@@ -42,6 +42,16 @@
...
+                <exclusion>
+                    <groupId>org.springframework.boot</groupId>
+                    <artifactId>spring-boot-starter-logging</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-log4j2</artifactId>
         </dependency>
```
Comment (jswk) [inline s4e-backend/pom.xml:55]: Isn't this a dependency of sentry-log4j2 anyway? You could leave it out as in gs-gateway maybe

**Review by jswk — APPROVED** [review]
In general looks good. I'd consider removing this one dependency from s4e-backend.
Moreover, description in README would be nice.

**Review by wziajka — APPROVED** [review]

**Pull request merged**